### PR TITLE
Delete obsolete Dockerfile

### PR DESCRIPTION
**Pull request by jawnsy**
Signed-off-by: Jonathan Yu <jawnsy@redhat.com>

### CODE REVIEW

**Comment by smarterclayton** [comment]
LGTM [merge]

**Comment by smarterclayton** [comment]
Heh, you broke the test by removing it.  Will need to change the test too.

**Comment by jawnsy** [comment]
Oopsie. Didn't realize there'd be tests for something we don't use anymore :)

Removed the failed test. Will a new Jenkins build automatically run, or do we have to do something?

File: test/cmd/builds.sh [hunk 1]
```diff
@@ -25,8 +25,6 @@ os::test::junit::declare_suite_start "cmd/builds"
 
 os::cmd::expect_success 'oc new-build centos/ruby-22-centos7 https://github.com/openshift/ruby-hello-world.git'
 os::cmd::expect_success 'oc get bc/ruby-hello-world'
-os::cmd::expect_success 'cat "${OS_ROOT}/Dockerfile" | oc new-build -D - --name=test'
```
Comment (pweil-) [inline test/cmd/builds.sh]: @bparees @csrwng should we be replacing this with a different dockerfile to keep the test or removing the test altogether?

Comment (csrwng) [inline test/cmd/builds.sh]: the test is useful, we should keep it ... just point it to a different Dockerfile, thx for catching that

Comment (csrwng) [inline test/cmd/builds.sh]: @jawnsy can you update your PR? you can use `${OS_ROOT}/images/origin/Dockerfile`

I tried creating a PR to fix it, but just realized this one hasn't merged.

Comment (jawnsy) [inline test/cmd/builds.sh]: Done, thanks!

**Comment by bparees** [comment]
lgtm [merge]

**Comment by jawnsy** [comment]
The failure output looks unrelated, is it a flake? Can someone confirm?

```
test/cmd/admin.sh:302: executing 'oc get ds/docker-registry --template="{{.status.desiredNumberScheduled}}"' expecting success and text '1'
```

**Comment by smarterclayton** [comment]
@derekwaynecarr does this seem familiar?

**Comment by derekwaynecarr** [comment]
not to me

**Comment by jawnsy** [comment]
Hm, might be a real failure. Not sure how these changes could've caused it, though.

> I0622 11:08:56.362794   19616 event.go:216] Event(api.ObjectReference{Kind:"DaemonSet", Namespace:"cmd-admin", Name:"docker-registry", UID:"3cf3dc5d-388b-11e6-ad46-0e2d04307177", APIVersion:"extensions", ResourceVersion:"548", FieldPath:""}): type: 'Warning' reason: 'FailedCreate' Error creating: pods "docker-registry-" is forbidden: service account cmd-admin/registry was not found, retry after the service account is created

**Comment by jawnsy** [comment]
Rebasing and re-running the tests seemed to do the trick, so I guess it was just a flake.

**Comment by smarterclayton** [comment]
[merge]

**Comment by bparees** [comment]
[merge]

**Pull request merged**